src/components/SpeechBubble.vue

```diff
@@ -24,9 +24,8 @@ function acknowledge() {
 div {
   max-width: 250px;
   position: absolute;
-  top: 45%;
-  left: -5%;
-  //transform: translateY(-50%);
+  top: 0;
+  transform: translate3D(175px, 200px, 510px) scale(0.5);
```

Comment:
The top of the bubble used to be nicely aligned with the TV. Now it moves around depending on the shown event. I suspect due to the changing size of the content of the bubble.

Reply:
Surprisingly: The longer the text in the bubble the further down the top of the bubble is rendered. I have no clue, why...

Reply:
Yes, that is very strange, and I have no solution for this yet, although I'm working on it.
I assume that it would be better to position the bubble absolutely in the 2D space above the scene. But there is still the problem that depending on the window ratio, the position cannot be simply defined by percentages of the window width and height...

Reply:
looking good!